supervisor/shared/serial.c

```diff
@@ -36,6 +36,12 @@
 
 #include "tusb.h"
 
+#ifdef NRF_DEBUG_PRINT
+// XXX  these functions are in nrf/supervisor/debug_uart.c
+extern void _debug_uart_init(void);
+extern void _debug_print_substr(const char* text, uint32_t length);
+#endif
```

Comment:
I think port specific code (even if it is guarded) should not be in the shared parts of the codebase.

Reply:
There could be an nrf-only include file that is conditionally included in the appropriate nrf-only files.

Reply:
I agree we should fix this, but let's open a new issue to do that, so this is not blocking the other sleep PR's